src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -373,3 +375,75 @@ message Version {
   // incremented if some job or task entity is created/updated.
   // uint64 revision = 2;
 }
+
+// BasicContainer stores the minimal data required to declare extra containers
+// to a job. This is in contrast to the Container message, which has other data
+// that are not strictly tied to the main container. For example,
+// *resources* (ram/cpu/etc) for the whole *task* are declared in the main
+// Container message, not in a basic container.
+message BasicContainer {
```

Comment:
Similar to volumes, `titus_container.proto`?

Reply:
Ok I've done this, but this will be a Hard breaking change here, because I can't take `Image` with me, so I've made a copy called `BasicImage`. 

Nobody except TitusAPI is using this proto, so it won't be super bad, but it will require an update to titus-control-plane and the functional tests.